src/db/migrations/31-add-relics-season-1.ts

```diff
@@ -0,0 +1,48 @@
+import { Knex } from 'knex';
+
+import { AudioUrlExtractorTypes, IdExtractorTypes, TitleExtractorTypes } from '../../types/fieldExtractor';
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { addNftFactory, addPlatform, removeNftFactory, removePlatform } from '../migration-helpers';
+
+const RELICS_YXZ: MusicPlatform = {
+  id: 'relics',
+  type: MusicPlatformType['multi-track-multiprint-contract'],
+  name: 'relics',
+}
+
+const SEASONS_1: NftFactory = {
+  id: '0x441C1266E6fb13C38c2752eab0D11A99905FFef4',
+  startingBlock: '14180427',
+  platformId: 'relics',
+  contractType: NFTContractTypeName.default,
+  standard: NFTStandard.ERC721,
+  autoApprove: true,
+  approved: true,
+  typeMetadata: {
+    overrides: {
+      artist: {
+        // TODO: add extractor for artist ID?
```

Comment:
yeah, there doesn't seem to be any great way to do it with relics yet, like an artist address or such, but I do think we should separate artists for relic in this PR nonetheless.

i guess best option for relics atm looks like:
 - artistNameExtractor that gets the Musician trait
 - artistIdExtractor that returns something like `${nftFactory.platformId}/${artistNameExtractor(nft)}`
 - websiteURL just https://relics.xyz/ for all for now?
 - avatarUrl all the same for now, as is?